src/t_stream.c

```diff
@@ -2389,8 +2590,8 @@ void streamDelConsumer(streamCG *cg, streamConsumer *consumer) {
  * Consumer groups commands
  * ----------------------------------------------------------------------- */
 
-/* XGROUP CREATE <key> <groupname> <id or $> [MKSTREAM]
- * XGROUP SETID <key> <groupname> <id or $>
+/* XGROUP CREATE <key> <groupname> <id or $> [MKSTREAM] [ENTRIESADDED entries_added]
+ * XGROUP SETID <key> <groupname> <id or $> [entries_added]
```

Comment:
i feel that we SHOULD add the ENTRIESADDED modifier in all commands that can take it (same with max_deleted_entry_id, i.e. add MAXDELETEDENTRYID)